lisa/conf.py

```diff
@@ -415,6 +447,9 @@ class MultiSrcConfMeta(abc.ABCMeta):
     template with the ``{generated_help}`` placeholder replaced by a snippet of
     ResStructuredText containing the list of allowed keys.
 
+    It will also create the types specified using ``newtype`` in the
+    :class:`KeyDesc`, along with a getter to expose it to ``exekall``.
```

Comment:
Might want to document here (or somewhere else - anywhere as long as it's documented!) that it's exposed as a camelcased attribute.

Reply:
It was shortly documented in KeyDesc docstring, but is now more explicit.